src/legacy/core_plugins/console/public/np_ready/application/components/top_nav_menu.tsx

```diff
@@ -29,15 +29,22 @@ export interface TopNavMenuItem {
 }
 
 interface Props {
+  disabled: boolean;
   items: TopNavMenuItem[];
 }
 
-export function TopNavMenu({ items }: Props) {
+export function TopNavMenu({ items, disabled }: Props) {
```

Comment:
Again, my preference is as follows. Partly to be explicit, and partly so TS will should at me if I accidentally try to return something that React can't render 🙈. It's up to you, of course.

```suggestion
export const TopNavMenu: FunctionComponent<Props> = ({ items, disabled }) => {
```

Reply:
Right, I had not thought of it that way! Happy to change it.

However does TS not complain about this at the call site anyway?